lib/core/src/Cardano/Wallet.hs

```diff
@@ -604,15 +604,10 @@ restoreBlocks ctx wid blocks nodeTip = db & \DBLayer{..} -> do
                 liftIO $ logCheckpoint cp'
                 putCheckpoint (PrimaryKey wid) cp'
 
-        liftIO $ logCheckpoint cp
+        liftIO $ logCheckpoint (NE.last cps)
```

Comment:
So it would `Creating checkpoint at` the existing checkpoint instead of the new? 🙃 👍

Reply:
No it wouldn't :) 
The next line (doing the insert) was fine, but the log line was off.

Reply:
Ah, yes, implicitly meant _log_ `"Creating checkpoint at"`.